src/_components/guides/in_depth/post_info.rb

```diff
@@ -0,0 +1,22 @@
+class Guides::InDepth::PostInfo < Bridgetown::Component
+  def initialize(authors:, last_update:)
+    @authors = authors.class == String ? [authors] : authors
```

Comment:
```suggestion
    @authors = authors.is_a?(String) ? [authors] : authors
```

Reply:
This switch is not a good practice. We often don't have choice. 
May be you can write: 
```suggestion
    @authors = Array(authors)
```
<img width="358" alt="image" src="https://github.com/user-attachments/assets/8119be2a-f17c-46d9-be77-925698c92f32">